<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiaryEditor</class>
 <widget class="QWidget" name="DiaryEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>531</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>599</width>
      <height>529</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>511</height>
      </rect>
     </property>
     <property name="title">
      <string>日志</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>标题</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titleEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>正文</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="contentEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>501</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="handinBtn">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancleBtn">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>作者：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>450</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
